README.md

```diff
@@ -132,6 +132,33 @@ make
 
 # build and then start the query REST frontend
 ../bin/start_clipper.sh
+
+```
+
+__Using Vagrant:__
+
+You could use vagrant to automatically create a development environment. Vagrant
+will create a virtual machine containing this environment. To use
+that approach you need to install both vagrant and virtualbox. Once that is
+done, just do the following commands to initialize vagrant.
```

Comment:
Some minor grammar edits:

"You can use Vagrant to automatically create a development environment in a virtual machine. This is one method for using Windows for Clipper development. Before you start, you must have [Vagrant](https://www.vagrantup.com/) and [VirtualBox](https://www.virtualbox.org/wiki/VirtualBox) installed. Once you have them installed, perform the following command to initialize Vagrant."